source/stylesheets/site.css.scss

```diff
@@ -18,115 +18,8 @@ body, html {
   text-align: center;
 }
 
-.font-size {
-  text-align: right;
-  font-family: 'Libre Baskerville', serif;
-  font-size: 11px;
-  padding: 1em;
-}
-
-.title {
-  font-family: 'Bitter', serif;
-  text-align: center;
-  margin: 5px lightgrey;
-}
-
-.text {
-  font-family: 'Libre Baskerville', serif;
-}
+@import '_logo.css.scss';
```

Comment:
@EnigmaEdivad Vedo che ha funzionato lo stesso ma puoi evitare di inserire `_` all'interno dell'`import`,
Deve essere presente solo nel nome del file di destinazione.
Quindi qui potevi limitarti a scrivere:

```css
@import 'logo.css.scss';
```

Reply:
Non è comunque bloccante.
Per me la issue è OK.

Reply:
procedo al merge quindi?

Reply:
Go for it! 👍